pkg/helpers/fixtures.go

```diff
@@ -26,6 +26,16 @@ func GetMockAppList() []models.App {
 	return apps
 }
 
+//GetMockApp returns a dummy app
+func GetMockApp() *models.App {
+	app := &models.App{
+		ID:      "7f89ce49-a736-459e-9110-e52d049fc025",
+		AppID:   "com.aerogear.mobile_app_one",
+		AppName: "Mobile App One",
+	}
+	return app
+}
+
```

Comment:
If your endpoint is to return the versions of some app, should not the mock has app + versions?

Reply:
@camilamacedo86 that would couple this function to the endpoint. The idea is that this function can be used throughout the application and at times we won't need to return the versions of the app. What if we want to mock an app that does not have any versions?

Reply:
@craicoverflow, I understand that this endpoint will be used in the detailed page and should return the versions of the app. ( PS.: Yes I do not agree with it when start and I thought that we should have a specific endpoint for the versions, however, I understand that the propose defined for this endpoint is return app + versions ) so, we should are mocking and testing the main idea of it.

Reply:
Ah @camilamacedo86 yes I understand you now. Yes the test file should also mock the app versions.

Reply:
Shows that return app + versions instead of a red/update for each have been struggling us to do it works and covers all scenarios. :-(

Reply:
@camilamacedo86 good point will do something like
```go
//GetMockApp returns a dummy app
func GetMockApp() *models.App {
	versionlist := GetMockAppVersionList()
	app := &models.App{
		ID:      "7f89ce49-a736-459e-9110-e52d049fc025",
		AppID:   "com.aerogear.mobile_app_one",
		AppName: "Mobile App One",
		DeployedVersions: &versionlist,
	}
	return app
}
```